### PR TITLE
Add inactive team prompt email

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add a prompt to teams which are considered inactive (definition undefined), and offer them some help.

Yet to work out what 'inactive' means specifically (will be based on IDE / Forum activity and number of competitor accounts), so the wording might change slightly once we've found numbers.

### CODE REVIEW

**Comment by antoinepetty** [comment]
Copypaste error in subject "Southampton Tech Day 8th Feb"

File: SR2020/2020-02-19-inactive-team-prompt.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+to: Teams deemed 'inactive'
+subject: Lack of activity in Student Robotics
+---
+
+Hello There!
+
+Our records show you've not been especially active on the IDE or Forum recently, and we want to offer our support!
```
Comment (PeterJCLaw) [inline SR2020/2020-02-19-inactive-team-prompt.md]: How are we planning to get the forum data?
We probably can get that, though it's not something we've got an existing script for. Would be a good one to add to the toolbox though.

File: SR2020/2020-02-19-inactive-team-prompt.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+to: Teams deemed 'inactive'
+subject: Lack of activity in Student Robotics
```
Comment (PeterJCLaw) [inline SR2020/2020-02-19-inactive-team-prompt.md]: Is there a wording which sounds more friendly?

Comment (RealOrangeOne) [reply SR2020/2020-02-19-inactive-team-prompt.md]: This was the nicest I could come up with, albeit with very little thinking.

File: SR2020/2020-02-19-inactive-team-prompt.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2020/2020-02-19-inactive-team-prompt.md]: ah, I'd forgotten we don't track forum activity, will amend

Comment (trickeydan) [reply SR2020/2020-02-19-inactive-team-prompt.md]: We could potentially write a phpBB extension for this. I'd potentially be willing to give it a shot depending on how soon you need it.

Comment (RealOrangeOne) [reply SR2020/2020-02-19-inactive-team-prompt.md]: The IDE is a far better and more useful metric right now for team participation. In hindsight, we have a lot of teams which never use the IDE and yet are very compitent and active teams, who likely wouldn't need chasing.

That being said, i'm sure there are better metrics than simply the IDE / number of competitors

File: SR2020/2020-02-19-inactive-team-prompt.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+to: Teams deemed 'inactive'
+subject: Student Robotics inactivity
```
Comment (prophile) [inline SR2020/2020-02-19-inactive-team-prompt.md]: This subject line is pretty blunt. This would make me less likely to become active.

**Comment by RealOrangeOne** [comment]
This has been sent to the 6 teams deemed 'inactive', as discussed in Slack.

**Pull request merged**